puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/vars/foreman/1.23.groovy

```diff
@@ -14,5 +14,7 @@ def foreman_server_distros = [
     'ubuntu1604',
     'ubuntu1804'
 ]
-
+def foreman_supported_dists = [
```

Comment:
Perhaps this should be `foreman_redhat_releases` to match the debian counter part?